src/views/Home.vue

```diff
@@ -64,9 +64,16 @@
             <ion-button slot="end" fill="outline" size="medium">View all</ion-button>
         </ion-item>
   
-        <div class="scroller-content">
-          <div class="scroller-item" v-for="i = 1 in 10" :key="i">  
-              <ion-card>
+        <div class="scroller-wrapper" ref="ionCard">
```

Comment:
Did we use this ref anywhere? If no, then please remove this ref.

Reply:
Yeah we have used